src/components/paginaFornecedor.js

```diff
@@ -0,0 +1,253 @@
+import React, { Component } from "react";
+import logo from "../img/Logo.png";
+import styled from "styled-components";
+import { Button, Typography } from "@material-ui/core";
+import SearchIcon from "@material-ui/icons/Search";
+import axios from "axios";
+// import { FilterNone } from "@material-ui/icons";
+import Rodape from "./Rodape";
+
+const NavWrapper = styled.nav`
+  display: flex;
+  align-items: center;
+  justify-content: space-evenly;
+
+  background: #f3940c;
+  li {
+    list-style: none;
+    margin: 1.2rem;
+    color: white !important;
+    font-size: 1.3rem;
+    text-transform: capitalize;
+    &:hover {
+      cursor: pointer;
+      text-decoration: underline;
+    }
+  }
+`;
+const NavLogo = styled.img`
+  height: 10vh;
+  width: 5vw;
+  margin: 1.2rem;
+`;
+const SearchBar = styled.div`
+  display: flex;
+  width: 50%;
+`;
+const SearchInput = styled.input`
+  border-radius: 5px;
+  width: 100%;
+  border-color: white;
+  outline: none;
+`;
+const SearchBtn = styled.button`
+  position: relative;
+  background-color: transparent;
+  outline: none;
+  border: none;
+  &:hover {
+    cursor: pointer;
+  }
+`;
+
+export default class Navbar extends Component {
+  state = {
+    dadosMeuBanco: [],
+    nome: "",
+    descricao: "",
+    preco: "",
+    metodoPgto: "",
+    categoria: "",
+    fotos: "",
+    numeroParcelas: "",
+  };
+
+  postarProdutos = () => {
+    const body = {
+      name: this.state.nome,
+      description: this.state.descricao,
+      price: Number(this.state.preco),
+      paymentMethod: this.state.metodoPgto,
+      category: this.state.categoria,
+      photos: [this.state.fotos],
+      installments: Number(this.state.numeroParcelas),
+    };
+
+    axios
+      .post(
+        "https://us-central1-labenu-apis.cloudfunctions.net/eloFourOne/products",
+        body
+      )
+      .then((response) => {
+        console.log("res", response);
+        this.setState({ nome: "" });
+        this.setState({ descricao: "" });
+        this.setState({ preco: "" });
+        this.setState({ metodoPgto: "" });
+        this.setState({ categoria: "" });
+        this.setState({ fotos: "" });
+        this.setState({ numeroParcelas: "" });
```

Comment:
Pra setar vários estados de uma vez, você não precisa criar um setState para cada!
Se fizessem:
`this.setState({nome: "", descricao: "", preco:""})` ... e etc até o fim daria certo também

Acho que no caso de sucesso, também seria legal mostrar um alert pro usuário dizendo que o produto dele foi adicionado e, no caso de erro, mostrar um alert com uma mensagem de erro